middleware/markdown/summary/render.go

```diff
@@ -9,23 +9,26 @@ import (
 // Ensure we implement the Blackfriday Markdown Renderer interface
 var _ blackfriday.Renderer = (*Renderer)(nil)
 
+// Renderer is a plain-text Markdown renderer that implements the
+// blackfriday.Renderer interface.  Many of the required methods are
+// stubs with no output.
 type Renderer struct{}
```

Comment:
```go
// Renderer is a plain-text Markdown renderer that implements the
// blackfriday.Renderer interface.
// 
// Many of the methods are stubs with no output to prevent output of HTML markup.
```